.ci/scripts/dswx_compare_opera_pge.py

```diff
@@ -0,0 +1,223 @@
+        return False
+
+    print(f'input 1 is {file_1}')
+    print(f'input 2 is {file_2}')
+
+    flag_all_ok = True
+
+    # TODO: compare projections ds.GetProjection()
+    layer_gdal_dataset_1 = gdal.Open(file_1, gdal.GA_ReadOnly)
+    geotransform_1 = layer_gdal_dataset_1.GetGeoTransform()
+    metadata_1 = layer_gdal_dataset_1.GetMetadata()
+    nbands_1 = layer_gdal_dataset_1.RasterCount
+
+    layer_gdal_dataset_2 = gdal.Open(file_2, gdal.GA_ReadOnly)
+    geotransform_2 = layer_gdal_dataset_2.GetGeoTransform()
+    metadata_2 = layer_gdal_dataset_2.GetMetadata()
+    nbands_2 = layer_gdal_dataset_2.RasterCount
+
+    # compare number of bands
+    flag_same_nbands =  nbands_1 == nbands_2
+    flag_all_ok, flag_same_nbands_str = _get_prefix_str(flag_same_nbands, flag_all_ok)
+    prefix = PREFIX
+    print(f'{flag_same_nbands_str}Comparing number of bands')
+    if not flag_same_nbands:
+        print(prefix + f'Input 1 has {nbands_1} bands and input 2'
+              f' has {nbands_2} bands')
+    else:
+        # compare each band
+        print('Comparing DSWx bands...')
+        for b in range(1, nbands_1 + 1):
+            gdal_band_1 = layer_gdal_dataset_1.GetRasterBand(b)
+            gdal_band_2 = layer_gdal_dataset_2.GetRasterBand(b)
+            image_1 = gdal_band_1.ReadAsArray()
+            image_2 = gdal_band_2.ReadAsArray()
+            is_close_result = np.isclose(image_1, image_2,
+                                         atol=COMPARE_DSWX_HLS_PRODUCTS_ERROR_TOLERANCE_ATOL,
+                                         rtol=COMPARE_DSWX_HLS_PRODUCTS_ERROR_TOLERANCE_RTOL)
+            num_image_differences = np.sum(~is_close_result)
+            if num_image_differences > 0:
+                flag_bands_are_equal = False
+                is_close_differences = abs( image_1[~is_close_result] - image_2[~is_close_result])
+                max_difference_band = np.max(is_close_differences)
+                min_difference_band = np.min(is_close_differences)
+
+                print(prefix + f"* image band {b} difference count is {num_image_differences} "
+                      f"with maximum difference of {max_difference_band} and minimum difference of "
+                      f"{min_difference_band}.")
+                differing_indices = np.where(~is_close_result)
+                i, j = differing_indices[0][0], differing_indices[1][0]
+                print(prefix +
+                      f'* e.g. image band {b} input 1 has value "{image_1[i, j]}" in position'
+                      f' (i: {i}, j: {j}) whereas input 2 has value "{image_2[i, j]}"'
+                      ' in the same position.')
+            else:
+                flag_bands_are_equal = True
+
+            flag_all_ok, flag_bands_are_equal_str = _get_prefix_str(flag_bands_are_equal,
+                                                       flag_all_ok)
+            print(f'{flag_bands_are_equal_str}     Band {b} -'
```

Comment:
This has some extra spaces that mis-aligns the `"Band {b}"` part of the message.

Given my comment up above about removing the `"Comparing DSWx bands..."` line, replace those spaces with `"Comparing "`, so that it reads like one of these:
```
[FAIL] Comparing Band 1 - Water classification (WTR)
[OK]   Comparing Band 1 - Water classification (WTR)
```

At the very least, remove the extra extra spaces, so that `"Band {b}"` aligns with the rest of the output.